packages/@o3r/third-party/schematics/index.it.spec.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Test environment exported by O3rEnvironment, must be first line of the file
+ * @jest-environment @o3r/test-helpers/jest-environment
+ * @jest-environment-o3r-app-folder test-app-application
+ */
+const o3rEnvironment = globalThis.o3rEnvironment;
+
+import {
+  addImportToAppModule,
+  getDefaultExecSyncOptions,
+  getGitDiff,
+  packageManagerExec,
+  packageManagerInstall,
+  packageManagerRunOnProject
+} from '@o3r/test-helpers';
+import * as path from 'node:path';
+
+describe('new Angular application', () => {
+  test('should add Otter Third Party to existing Angular app', async () => {
+    const { applicationPath, workspacePath, libraryPath, isInWorkspace, appName, o3rVersion, untouchedProjectsPaths } = o3rEnvironment.testEnvironment;
+    const execAppOptions = {...getDefaultExecSyncOptions(), cwd: workspacePath};
+    const relativeApplicationPath = path.relative(workspacePath, applicationPath);
+
+    packageManagerExec({script: 'ng', args: ['add', `@o3r/third-party@${o3rVersion}`,'--project-name', appName, '--skip-confirmation']}, execAppOptions);
+    expect(() => packageManagerInstall(execAppOptions)).not.toThrow();
+
+    const diff = getGitDiff(workspacePath);
+    expect(diff.modified).toContain('package.json');
+    expect(diff.modified).toContain('angular.json');
+    expect(diff.modified).toContain('apps/test-app/package.json');
+    expect(diff.modified.length).toBe(4); // with yarn.lock or package-lock.json
```

Comment:
we can check for the actual file to be created https://github.com/AmadeusITGroup/otter/pull/1995/files#diff-e0410f3a4b89d7e26e783977c7180b91cb6d2b6e9587a36bd252dc4e4157616dR59